homeassistant/components/openuv/__init__.py

```diff
@@ -55,54 +52,12 @@
 TYPE_SAFE_EXPOSURE_TIME_5 = "safe_exposure_time_type_5"
 TYPE_SAFE_EXPOSURE_TIME_6 = "safe_exposure_time_type_6"
 
-
-CONFIG_SCHEMA = vol.Schema(
```

Comment:
Should we mark it as deprecated for now so people know the configuration doesn't work ? 

```python
CONFIG_SCHEMA = cv.deprecated(DOMAIN, invalidation_version="0.115")
```